### ISSUE TITLE
[javascript] unnecessary block for try/catch

### ISSUE THREAD

**Issue by dah-fari7009** [pmd/pmd#4055]
**Affects PMD Version:** 
6.47.0

**Rule:**
UnnecessaryBlock
<https://pmd.github.io/latest/pmd_rules_ecmascript_codestyle.html#unnecessaryblock>

**Description:**
Same problem as #4054 
PMD reports an unnecessary block for the try/catch, but there's no reason why this should be the case. Further, looking into the generate AST by PMD, it seems to be an issue with handling async/await keywords. In fact, without:
![image](https://user-images.githubusercontent.com/35702512/179899775-379babe3-30c6-4beb-a588-38fb9ef6e0b2.png)

and with it, it shows EmptyStatement instead:
![image](https://user-images.githubusercontent.com/35702512/179899922-bca3501a-dc79-425f-91b5-7649e11fbd1a.png)

**Code Sample demonstrating the issue:**

```
async function run(){
    try{
        await client.connect()
        console.log("Successfully connected to the database")
        var server = app.listen(8081, function (){
            var host = server.address().hostname
            var port = server.address().port
            console.log("Example app running at https://%s:%s", host, port)
        })
    }
    catch(err){
        console.log(err)
        await client.close()

    }
}
```

**Expected outcome:**

PMD reports a violation at line 2 for the try block, but that seems to be a false positive.

**Running PMD through:** *[Designer]*

**Comment by jsotuyod** [comment]
Even if `await` / `async` is still unsupported by Rhino, this no longer produces a FP under PMD 7.0.0
